paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -120,6 +120,15 @@ class MKLDNNHandlerT {
     return (dev_ctx_.GetBlob(key_p) != nullptr);
   }
 
+  bool isBwdCached() {
```

Comment:
Weird function name. The is no "isCached" function called inside.

Reply:
We will make up something better once we progress with refactoring.